### PR TITLE
Usama/blog/blog item

### PR DESCRIPTION
**Pull request by UzeeChamp1**
WORK IN PROGRESS

Blogs has been designde, Blogs are yet to be added

### CODE REVIEW

File: src/components/blog/blogs/blog-item.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import React from 'react';
```
Comment (alitjanjua) [inline src/components/blog/blogs/blog-item.tsx:1]: @UzeeChamp1 component name should start with Capital letter and shouldn't contain alphanumeric letters
```suggestion
src/components/blog/BlogItem.tsx
```

**Review by alitjanjua — CHANGES_REQUESTED** [review]

File: src/components/blog/blogs/blog-item.tsx (hunk 1 above)
Comment (UzeeChamp1) [reply src/components/blog/blogs/blog-item.tsx:1]: actually the component function name is already in camel case, this is just a file name, Should I change it too?

**Review by alitjanjua — APPROVED** [review]

File: pages/blogs.tsx [hunk 2]
```diff
@@ -1,13 +1,17 @@
 import * as React from 'react';
-import { Box } from '@mui/material';
+import { Box, Container } from '@mui/material';
```
Comment (alitjanjua) [inline pages/blogs.tsx:2]: @UzeeChamp1 when you have unused imports, pls remove them

File: src/components/blog/blogs/BlogItem.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  blogItemBtn: {
+    padding: 0,
+    color: '#000',
+    fontSize: '12px',
+    textTransform: 'none',
+    '@media (max-width: 600px)': {},
+  },
+});
+
+interface BlogItemProps {
```
Comment (alitjanjua) [inline src/components/blog/blogs/BlogItem.tsx:67]: @UzeeChamp1 pls start the interface name with capital i(I)
```suggestion
interface IBlogItemProps {
```

File: src/components/blog/blogs/index.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  loadMoreBtn: {
+    color: 'black',
+  },
+});
+
+export function BlogList() {
+  const classes = useStyles();
+  const [loading, setLoading] = useState(false);
+  const [blogsToShow, setBlogsToShow] = useState(9);
+  const handleLoadMore = () => {
```
Comment (alitjanjua) [inline src/components/blog/blogs/index.tsx:31]: @UzeeChamp1 when you start writing functions in component, make sure to add empty line before them
```suggestion

  const handleLoadMore = () => {
```

**Pull request merged**